app/templates/frameworks/edit_declaration_section.html

```diff
@@ -6,28 +6,29 @@
 {% endblock %}
 
 {% block breadcrumb %}
-  {%
-    with items = [
+  {{ govukBreadcrumbs({
+    "items": [
       {
-        "link": "/",
-        "label": "Digital Marketplace"
+        "text": "Digital Marketplace",
+        "href": "/"
       },
       {
-        "link": url_for(".dashboard"),
-        "label": "Your account"
+        "text": "Your account",
+        "href": url_for('.dashboard')
       },
       {
-        "link": url_for(".framework_dashboard", framework_slug=framework.slug),
-        "label": "Apply to " + framework.name
+        "text": "Apply to " + framework.name,
+        "href": url_for(".framework_dashboard", framework_slug=framework.slug)
       },
       {
-        "link": url_for(".framework_supplier_declaration_overview", framework_slug=framework.slug),
-        "label": "Your declaration"
+        "text": "Your declaration",
+        "href": url_for(".framework_supplier_declaration_overview", framework_slug=framework.slug)
+      },
+      {
+        "text": framework.name + " supplier declaration"
```

Comment:
I think this looks better with the section name

```suggestion
        "text": section.name
```